src/WaitingRoom.js

```diff
@@ -40,6 +49,21 @@ export default class WaitingRoom extends Component {
     firebase.database().ref("/session-metadata").off();
   }
 
+  onFilterGames(filterType) {
+    this.setState({ filterType });
```

Comment:
Shouldn't this be `this.setState({ filterType: filterType });`?

Reply:
Yeah - I just did object shorthand but can do it that way if it is more readable

Reply:
I didn't realize you could do that -- neat!